lib/garage/controller_helper.rb

```diff
@@ -34,7 +34,7 @@ def resource_owner_id
     # Use this method to render 'unauthorized'.
     # Garage user may overwrite this method to response custom unauthorized response.
     # @return [Hash]
-    def unauthorized_render_options
+    def unauthorized_render_options(error: nil)
```

Comment:
I mis-understood about wrapper. Patching garage-doorkeeper could be good. But since this render option is originally doorkeeper's idea, it is good garage follows new doorkeeper's idea.